packages/design-system-tokens/src/themes/core/defaultTheme.ts

```diff
@@ -131,6 +135,29 @@ export const components = {
   'badge__background-color--warn': themeColors.warn,
   'badge__font-color--warn': themeColors.base,
   'badge__border-radius': radius.pill,
+  // choice + choicelist
+  'choice__background-color': themeColors.background,
+  'choice__background-color--checked': themeColors.primary,
+  'choice__background-color--disabled': themeColors['gray-lighter'],
+  'choice__background-color--inverse': 'transparent',
+  'choice__background-color--disabled--inverse': hexOpacity(themeColors['muted-inverse'], 15),
+  'choice__background-color--unchecked': themeColors.primary,
+  'choice__border-color': themeColors.base,
+  'choice__border-color--checked': themeColors.primary,
+  'choice__border-color--disabled': themeColors['gray-light'],
+  'choice__border-color--error': themeColors.error,
+  'choice__border-color--inverse': themeColors.white,
+  'choice__border-color--left': themeColors.primary,
+  'choice__border-color--focus': themeColors['primary-darker'],
+  'choice__border-color--disabled--inverse': themeColors['gray-light'],
+  'choice__border-radius': '0px',
+  'choice__border-width': '2px',
+  'choice__color--unchecked': themeColors.white,
+  'choice__color--disabled': themeColors.muted,
+  choice__size: spacer['4'],
```

Comment:
Does this key not being in quotes impact anything?

Reply:
Good point, no it doesn't, more readable that way, updated.  I just got used to needing it with color vars.